app/models/pipeline_run.rb

```diff
@@ -844,9 +844,12 @@ def update_job_status
         self.job_status = STATUS_FAILED
         self.finalized = 1
         self.known_user_error, self.error_message = check_for_user_error(prs)
-        log_message = "SampleFailedEvent: Sample #{sample.id} by #{sample.user.admin? ? 'admin user' : 'non-admin user'} " \
-          "failed #{prs.name} with #{adjusted_remaining_reads || 0} reads remaining after #{duration_hrs} hours. " \
-          "See: #{status_url}"
+        log_message = "SampleFailedEvent: Sample #{sample.id} " \
+          "by #{sample.user.admin? ? 'admin user' : 'non-admin user'} " \
```

Comment:
nit: can you add the processing of this interpolated variables before log_message like ```
user_role = sample.user.admin? ? 'admin user' : 'non-admin user'
log_message = "[...] by #{user_role}"
```
Just makes it more readable...